test/test_nn.py

```diff
@@ -8564,8 +8564,8 @@ def test_nnpack_conv(self):
                     weight = torch.randn(weight_shape, requires_grad=True, dtype=torch.float)
                     if has_bias:
                         bias = torch.randn([chan_out], requires_grad=True, dtype=torch.float)
-                    output = torch._nnpack_spatial_convolution(input, weight, padding=padding, bias=bias)
-                    output_expected = torch.nn.functional.conv2d(input, weight, padding=padding, bias=bias)
+                    output = torch._nnpack_spatial_convolution(input, weight, stirde=stride, padding=padding, bias=bias)
```

Comment:
do these tests pass? `stirde` should be `stride` right?

Reply:
Great find.  This should be at least part of the reason with these test failures.